Problem compiling on Ubuntu 20.10

I'm trying to compile on Ubuntu 20.10 but I got some problems, any tips ? (it was working on Ubuntu 20.04)

```
user@mypc:~/nfc-srix-tools/build$ make
make[1] : on entre dans le répertoire « /home/user/nfc-srix-tools/build »
make[2] : on entre dans le répertoire « /home/user/nfc-srix-tools/build »
make[2] : on quitte le répertoire « /home/user/nfc-srix-tools/build »
make[2] : on entre dans le répertoire « /home/user/nfc-srix-tools/build »
[  6%] Building C object CMakeFiles/srix-reset.dir/otp_reset.c.o
[ 12%] Building C object CMakeFiles/srix-reset.dir/logging.c.o
[ 18%] Building C object CMakeFiles/srix-reset.dir/nfc_utils.c.o
[ 25%] Linking C executable srix-reset
/usr/bin/ld : CMakeFiles/srix-reset.dir/logging.c.o:(.bss+0x0) : définitions multiples de « verbose_status »; CMakeFiles/srix-reset.dir/otp_reset.c.o:(.bss+0x0) : défini pour la première fois ici
/usr/bin/ld : CMakeFiles/srix-reset.dir/logging.c.o:(.bss+0x4) : définitions multiples de « verbosity_level »; CMakeFiles/srix-reset.dir/otp_reset.c.o:(.bss+0x4) : défini pour la première fois ici
/usr/bin/ld : CMakeFiles/srix-reset.dir/nfc_utils.c.o:(.rodata+0x0) : définitions multiples de « nmISO14443B »; CMakeFiles/srix-reset.dir/otp_reset.c.o:(.rodata+0x0) : défini pour la première fois ici
/usr/bin/ld : CMakeFiles/srix-reset.dir/nfc_utils.c.o:(.rodata+0x8) : définitions multiples de « nmISO14443B2SR »; CMakeFiles/srix-reset.dir/otp_reset.c.o:(.rodata+0x8) : défini pour la première fois ici
/usr/bin/ld : CMakeFiles/srix-reset.dir/nfc_utils.c.o:(.bss+0x0) : définitions multiples de « verbose_status »; CMakeFiles/srix-reset.dir/otp_reset.c.o:(.bss+0x0) : défini pour la première fois ici
/usr/bin/ld : CMakeFiles/srix-reset.dir/nfc_utils.c.o:(.bss+0x4) : définitions multiples de « verbosity_level »; CMakeFiles/srix-reset.dir/otp_reset.c.o:(.bss+0x4) : défini pour la première fois ici
collect2: error: ld returned 1 exit status
make[2]: *** [CMakeFiles/srix-reset.dir/build.make:114 : srix-reset] Erreur 1
make[2] : on quitte le répertoire « /home/user/nfc-srix-tools/build »
make[1]: *** [CMakeFiles/Makefile2:82 : CMakeFiles/srix-reset.dir/all] Erreur 2
make[1] : on quitte le répertoire « /home/user/nfc-srix-tools/build »
make: *** [Makefile:84 : all] Erreur 2
```

--- a/logging.h
+++ b/logging.h
@@ -27,8 +27,8 @@
 #define BOLD "\033[1m"
 #define DIM "\033[2m"
 
-bool verbose_status;
-int verbosity_level;
+extern bool verbose_status;
+extern int verbosity_level;
 void set_verbose(bool);
 void set_verbosity(int);
 int lverbose(const char * restrict, ...);

--- a/nfc_utils.h
+++ b/nfc_utils.h
@@ -30,8 +30,8 @@
 #define SR_WRITE_BLOCK_COMMAND 0x09
 
 /* Constants */
-const nfc_modulation nmISO14443B;
-const nfc_modulation nmISO14443B2SR;
+extern const nfc_modulation nmISO14443B;
+extern const nfc_modulation nmISO14443B2SR;
 
 /* Logging */
 void log_command_sent(const uint8_t *command, size_t num_bytes);

